sunpy/util/tests/test_decorators.py

```diff
@@ -5,42 +5,27 @@
 import astropy.units as u
 
 from sunpy.util.decorators import (
-    active_contexts,
+    ACTIVE_CONTEXTS,
     deprecate_positional_args_since,
     deprecated,
-    get_removal_version,
     sunpycontextmanager,
 )
-from sunpy.util.exceptions import SunpyDeprecationWarning, SunpyPendingDeprecationWarning
-
-
-def test_removal_version_since_lts():
-    major, minor = get_removal_version('2.0')
-    assert major == 2
-    assert minor == 1
-
-
-def test_removal_version_not_since_lts():
-    major, minor = get_removal_version('2.1')
-    assert major == 3
-    assert minor == 1
+from sunpy.util.exceptions import SunpyDeprecationWarning
 
 
 @pytest.mark.parametrize(
-    ('since', 'pending', 'warning', 'message', 'warning_message'),
+    ('since', 'warning', 'message', 'warning_message'),
     [
-        ('2.0', False, SunpyDeprecationWarning, '',
-         'The foo function is deprecated and may be removed in version 2.1.'),
-        ('2.1', False, SunpyDeprecationWarning, '',
-         'The foo function is deprecated and may be removed in version 3.1.'),
-        ('2.0', True, SunpyPendingDeprecationWarning, '',
-         'The foo function will be deprecated in version 2.0.'),
-        ('2.0', False, SunpyDeprecationWarning,
+        ('2.0', SunpyDeprecationWarning, '',
+         'The foo function is deprecated and may be removed in a future version'),
```

Comment:
This is definitely not ideal. How much of a pain is it to work around?

Reply:
on second thoughts let's sack it off, we should probably at least open an issue on astropy to support this though.

Reply:
I don't think it would THAT much work but I would need to sit down and do it.

It would be nice for astropy to support this, would they accept that PR?